interview-questions.md

```diff
@@ -8,50 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: a decision tree
 
-Researched answer:
+Researched answer: commands for handling decisions
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: global information tracker.
+git is on our computers.
+Github is located in Idaho
 
-Researched answer:
+Researched answer: created in 2005 by Linus Torvalds to track changes in source code.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: ==
```

Comment:
Interview answers need to be expanded in the future.  This meets the minimum expectations of the assessments.  These answers should reflect how you would respond in an interview.  Short one-liner answers will not impress interviewers.  Use definitions and examples to really share your knowledge.